tools/dockerbuild/docker/Dockerfile-server

```diff
@@ -20,7 +20,7 @@
 # (See site/docs/try/configuration.md)
 #
 ###
-FROM registry.access.redhat.com/ubi9/openjdk-21:1.18-4
+FROM registry.access.redhat.com/ubi9/openjdk-21-runtime:1.18-4
```

Comment:
After doing a `docker build & docker save` I observed that the total uncompressed image size went down from 682 MiB to 584 MiB. Nice!

I also validated that the `JAVA_OPTS_APPEND`,`JAVA_TOOL_OPTIONS` and `JAVA_DEBUG` env vars are still working.

Reply:
>  I observed that the total uncompressed image size went down from 682 MiB to 584 MiB. Nice!

Not so much on for amd64 - "just" down by 40MB - but still.